start-app.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-./gradlew build
+./gradlew build -x test
```

Comment:
maybe you want skipping detekt checks as well?

So, it's better to choose another task that just builds your app without extra checks.

```bash
./gradlew assemble
```  
Try this.


----

Remember, you can use option `--dry-run` that helps to understand task's dependendencies;
Try this two commands:
```bash
./gradlew build --dry-run
./gradlew assemble --dry-run 
```